docs/gathered-data.md

```diff
@@ -13,1040 +13,1878 @@ archives are packaged as `.tar.gz` files in `/var/lib/insights-operator`.
-Response see https://docs.openshift.com/container-platform/latest/rest_api/config_apis/image-config-openshift-io-v1.html#image-config-openshift-io-v1
+### Released version
+- 4.11.0
 
-* Location in archive: config/image.json
-* Id in config: clusterconfig/image
-* Since versions:
-  - 4.10.8
-  - 4.11+
+### Backported versions
+- 4.10.8+
+
+### Changes
+None
 
 
 ## ClusterImagePruner
 
-fetches the image pruner configuration
+Collects the image pruner configuration.
+
+### API Reference
+None
+
+### Sample data
+- [docs/insights-archive-sample/clusteroperator/imageregistry.operator.openshift.io/imagepruner/cluster.json](./insights-archive-sample/clusteroperator/imageregistry.operator.openshift.io/imagepruner/cluster.json)
+
+### Location in archive
+| Version   | Path																			     |
+| --------- | ----------------------------------------------------------------------------------- |
+| < 4.6.0   | config/imagepruner.json														 	 |
+| >= 4.6.0  | config/clusteroperator/{group}/{kind}/{name}.json |
+
+### Config ID
+`clusterconfig/image_pruners`
 
-* Location in archive: config/clusteroperator/imageregistry.operator.openshift.io/imagepruner/cluster.json
-* Location in older versions: config/imagepruner.json
-* Id in config: clusterconfig/image_pruners
+### Released version
+- 4.5.0
+
+### Backported versions
+None
+
+### Changes
+None
 
 
 ## ClusterImageRegistry
 
-fetches the cluster Image Registry configuration
+Collects the cluster Image Registry configuration
+
+### API Reference
+None
+
+### Sample data
+- [docs/insights-archive-sample/config/clusteroperator/imageregistry.operator.openshift.io/config/cluster.json](./insights-archive-sample/config/clusteroperator/imageregistry.operator.openshift.io/config/cluster.json)
+
+### Location in archive
+| Version   | Path																			|
+| --------- | ------------------------------------------------------------------------------ |
+| < 4.7.0   | config/imageregistry.json 					                        			|
+| >= 4.7.0  | config/clusteroperator/imageregistry.operator.openshift.io/config/cluster.json |
+| >= 4.6.20 | config/persistentvolumes/{name}.json 											|
+
+### Config ID
+`clusterconfig/image_registries`
 
-**Conditional data**: If the Image Registry configuration uses any PersistentVolumeClaim for the storage, the corresponding
-PersistentVolume definition is gathered
+### Released version
+- 4.5.0
 
-* Location in archive: config/clusteroperator/imageregistry.operator.openshift.io/config/cluster.json
-* Location in older versions: config/imageregistry.json
-* Id in config: clusterconfig/image_registries
-* Since versions:
-  - 4.3.40+
-  - 4.4.12+
-  - 4.5+
+### Backported versions
+- 4.3.40+
+- 4.4.12+
 
-* PV definition since versions:
-  - 4.6.20+
-  - 4.7+
+### Changes
+- "PersistentVolumes" were introduced in version 4.7.0 and backported to 4.6.20+.
```

Comment:
It hurts to see (at least) four different ways to denote gathered API objects throughout the document:

1. Name only, no extra formatting: Authentication
2. Name only, double-quoted: "PersistentVolumes"
3. Name only, mono-spaced: `PersistentVolumes`
4. Name and API group, mono-spaced: `cephclusters.ceph.rook.io`

I don't mean to bloat the scope of this PR - is it easier to unify it within this PR or create a separate one once this gets merged?

Reply:
I'll try to review it, but it is hard to get through all this stuff. There is no tool, you need to read very, very, very carefully and there is no tooling to help it, except your eyes.